shared/utils/UrlHelper.ts

```diff
@@ -0,0 +1,16 @@
+class Outline {
```

Comment:
Honestly lets lose the `Outline` namespace – we can assume it's the default…

Reply:
Sorry, you mean these should be accessible for instance, like `UrlHelper.twitter` instead of `UrlHelper.Outline.twitter`?

Reply:
yep